README.md

```diff
@@ -75,6 +75,14 @@ Badges look really great, but they're not always easy to set up. Take a look at
 
 <br>
 
+| **GitLab&nbsp;CI/CD** |  |
+|:--------------------------|:-|
+| Badge | [![Pipeline Status](https://gitlab.cn/Zalyoung/matlab-ci-example/badges/main/pipeline.svg)](https://gitlab.cn/Zalyoung/matlab-ci-example) |
+| Badge Code | \[\!\[Pipeline Status](https[]()://gitlab.cn/***GITLAB_PROJECT_PATH***/badges/***DEFAULT_BRANCH_NAME***/pipeline.svg)](https[]()://gitlab.cn/***GITLAB_PROJECT_PATH***) |
+| Badge Help | [GitLab CI/CD documentation for setting up badges](https://docs.gitlab.com/ee/user/project/badges.html "GitLab CI/CD documentation for setting up badges") |
```

Comment:
Same as above, it would be great to have a GitLab badge but we can't link out to a personal project. The badge code and help link are still useful though. Can we switch to `.com` however? Most of our users would be on `.com`.

```suggestion
| **GitLab&nbsp;CI/CD** |  |
|:--------------------------|:-|
| Badge Code | \[\!\[Pipeline Status](https[]()://gitlab.com/***GITLAB_PROJECT_PATH***/badges/***DEFAULT_BRANCH_NAME***/pipeline.svg)](https[]()://gitlab.com/***GITLAB_PROJECT_PATH***) |
| Badge Help | [GitLab CI/CD documentation for setting up badges](https://docs.gitlab.com/ee/user/project/badges.html "GitLab CI/CD documentation for setting up badges") |
```